cloud/src/definer/crud.definer.js

```diff
@@ -148,18 +148,42 @@ Parse.Cloud.define('postObjectsToClass', (request) => new Promise((resolve, reje
     localObject - Continas key value pairs that will be posted to the class
                 - this contains a latitude/longitude which will post the location
     parseParentClassID - ID of the parseParentClass Object to associate the new post with
+    loopParentID - Custom Form that the looped form originates from
+    loop - bool t/f whether looped data contained in form
   ******************************************* */
 Parse.Cloud.define('postObjectsToClassWithRelation', (request) => new Promise((resolve, reject) => {
   const supplementaryForm = new Parse.Object(request.params.parseClass);
   const residentIdForm = new Parse.Object(request.params.parseParentClass);
   const userObject = new Parse.Object('_User');
+  const loopParentForm = new Parse.Object(request.params.parseClass);
```

Comment:
are `loopParentForm` and `supplementaryForm` different?

Reply:
@hopetambala yes, the loopParentForm gets used when there is a loop used in the form (ie. one more loops on a form). This variable allows us to create a link between each of the form submissions when a loop is given. The main loop form can be considered the Parent like the SurveyData is the parent of supplementary forms.